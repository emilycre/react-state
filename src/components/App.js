import React from 'react';
import Increment from './Incrementer';

export default function App() {
  return <Increment />;
}
