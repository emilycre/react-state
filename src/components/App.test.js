import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App component', () => {
  it('Renders an app', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });

  it('Changes color on selectColor event', () => {
    const wrapper = shallow(<App />);

    wrapper.instance().selectColor('green');
    expect(wrapper.state('color')).toEqual('green');
  });
});
