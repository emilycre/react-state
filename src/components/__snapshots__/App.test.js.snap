// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Renders an App 1`] = `<RandomColor />`;
